src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -71,17 +71,18 @@ class RequestorStep extends React.Component {
      */
     validate() {
         if (!this.state.isControllingOfficer) {
-            Growl.error(this.props.translate('requestorStep.isControllingOfficerError'));
+            showErrorModal(this.props.translate('requestorStep.isControllingOfficerError'));
             return false;
         }
 
-        if (!isValidIdentity({
+        const identity = {
             street: this.state.requestorAddressStreet,
             state: this.state.requestorAddressState,
             zipCode: this.state.requestorAddressZipCode,
             dob: this.state.dob,
             ssnLast4: this.state.ssnLast4,
-        })) {
+        };
+        if (!isValidIdentity(identity)) {
```

Comment:
We can undo this change for now. Let's only make the changes we need to?

Reply:
done